crates/networking/rpc/net/mod.rs

```diff
@@ -0,0 +1,15 @@
+use serde_json::Value;
+
+use crate::{
+    utils::{RpcErr, RpcRequest},
+    RpcApiContext,
+};
+
+pub fn version(_req: &RpcRequest, context: RpcApiContext) -> Result<Value, RpcErr> {
+    let chain_spec = context
+        .storage
+        .get_chain_config()
+        .map_err(|error| RpcErr::Internal(error.to_string()))?;
+    serde_json::to_value(format!("{}", chain_spec.chain_id))
+        .map_err(|error| RpcErr::Internal(error.to_string()))
```

Comment:
Same here. Can't we just omit the map_err? The `From<serde_json::Error> for RpcErr` trait is implemented too.

Reply:
We can go about in 2 ways:

1) a shorter version of map_err: 
```
serde_json::to_value(format!("{}", chain_spec.chain_id))
        .map_err(RpcErr::from)
```

2) extract value and wrap it in Ok():
```
let value = serde_json::to_value(format!("{}", chain_spec.chain_id))?;
Ok(value)
```

Which is the preferred style señor?

Reply:
I went with 2)